log4j-core/src/test/java/org/apache/logging/log4j/core/config/ConfigurationSourceTest.java

```diff
@@ -30,4 +40,22 @@ public void testJira_LOG4J2_2770_byteArray() throws Exception {
         ConfigurationSource configurationSource = new ConfigurationSource(new ByteArrayInputStream(new byte[] { 'a', 'b' }));
         assertNotNull(configurationSource.resetInputStream());
     }
+
+    @Test
+    @EnabledOnOs({OS.LINUX, OS.AIX, OS.MAC, OS.SOLARIS})
```

Comment:
Why the restriction?

Reply:
Also I think we should keep prefixing test methods with "test" which makes it simpler to recognize in an IDE's method list view.

Reply:
I use `UnixOperatingSystemMXBean` to count file descriptors. This test would be useless on Windows.

Reply:
Because?